configuration/frontend/base-config.json

```diff
@@ -31,5 +31,17 @@
       "generalPatientNoteUuid": "3cd9d956-26fe-102b-80cb-0017a47871b2",
       "midUpperArmCircumferenceUuid": "4326b04b-3158-417a-bb8d-ad022295b0f4"
     }
+  },
+  "@openmrs/esm-patient-medications-app": {
+    "durationUnitsConcept": "3cd706b8-26fe-102b-80cb-0017a47871b2",
+    "daysDurationUnit": {
+      "uuid": "3cd706b8-26fe-102b-80cb-0017a47871b2"
+    },
+    "careSettingUuid": "6f0c9a92-6f24-11e3-af88-005056821db0",
+    "quantityUnitsUuid": "162395AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
```

Comment:
On humci, this is the concept uuid for "Bag".  So I believe this is the incorrect uuid. Which also adds evidence that we should not need to configure this here at all.  I'm also not sure what "quantityUnits" refers to.  There are global properties (and associated services) for core that are the following:  `order.drugDosingUnitsConceptUuid` and `order.drugDispensingUnitsConceptUuid`.  Can we not just get this from the core configuration, using the one(s) that are most appropriate (assuming this is intending to represent one of these?)